### DISCUSSION TITLE
Styling question for Menu on Search w/ AutoComplete

### DISCUSSION THREAD

**Discussion by ninavizz** [adobe/react-spectrum#7308] [General]
Hello! Is it possible to <strike>add additional stuff to the Menu items for Search w/ AutoComplete? Items that would be attached to the queried subjects.</strike> use the [ListBox component](https://react-spectrum.adobe.com/react-spectrum/ListBox.html), for displaying dynamic search results with AutoComplete Search?

In the below example, I'd like to be able to query on the last-4 of a person's SSN, or on their name. Likewise, I'd also like to display each in the Menu with their associated Local union office. 

LMK, thx! :)

<img width="338" alt="Screenshot 2024-11-02 at 3 45 06 PM" src="https://github.com/user-attachments/assets/1f13ee2c-3170-4b0d-b5c9-115bc0bd1802">

**Comment by reidbarber** [comment]
https://github.com/adobe/react-spectrum/pull/7181 may be what you're looking for.

**Reply by cheema-corellian** [reply]
@reidbarber 7308 is the ID of this discussion thread.

**Reply by reidbarber** [reply]
Sorry about that! Updated to https://github.com/adobe/react-spectrum/pull/7181
